assets/src/settings/index.js

```diff
@@ -0,0 +1,93 @@
+/**
```

Comment:
Do we actually need a Settings page? And if so, couldn't it just be good old-fashioned PHP-rendered output? Seems like having a React component with a corresponding API endpoint is overkill.